### ISSUE TITLE
Crash when loading images on native macOS target

### ISSUE THREAD

**Issue by zacharee** [icerockdev/moko-resources#489]
I'm experimenting with the macOS target with a JetBrains Compose cross-platform app and I'm running into a crash. I'm not exactly sure of the reproduction steps, but I've attached the stacktrace.

Project source: https://github.com/zacharee/ArcadyanKVD21Control.

- Kotlin 1.8.10.
- Compose 1.5.0-dev1049.
- Moko 0.22.0.

[Stacktrace.txt](https://github.com/icerockdev/moko-resources/files/11526323/Stacktrace.txt)

**Comment by Alex009** [comment]
error log:
```
*** -redComponent not valid for the NSColor Catalog color: #$assets-dev.zwander.resources.common.MR icon_color_one; need to first convert colorspace.
libc++abi: terminating due to uncaught exception of type NSException
```

**Comment by Alex009** [comment]
will be released in 0.22.3
